.gitignore

```diff
@@ -32,6 +32,7 @@ tests/data/common_wheels/
 *~
 .*.sw?
 .env/
+venv
```

Comment:
We should probably update this entire list with what's in GitHub's Python.gitignore template, in a separate PR.

By any chance, could you remove this change from this PR? That'll make this PR do exactly 1 thing, as described by the title.